README.md

```diff
@@ -26,7 +26,10 @@ key parts:
     RBI output file.
 
   - You can **effortlessly build tools which need to access types within an RBI**;
-    no need to write your own parser! 
+    no need to write your own parser!
+
+  - You can **generate RBI to ship with your gem** for consuming projects to use
+    ([see "RBIs within gems" in sorbet the docs](https://sorbet.org/docs/rbi#rbis-within-gems)).
```

Comment:
Very pedantic, but best to keep capitalisation consistent, and maybe tidy up the wording a bit here

```suggestion
    ([see "RBIs within gems" in Sorbet's docs](https://sorbet.org/docs/rbi#rbis-within-gems)).
```